### DISCUSSION TITLE
register my domain

### DISCUSSION THREAD

**Discussion by mastodonLike** [mastodon/mastodon#20176] [General]
Hi, I have created a domain name etc.., how can I register my domain with mastodon please ?

**Comment by MrGeneration** [comment]
You setup and configure your instance and you're done. You don't have to register your domain at all.
It works natively - you'll just have to find people on e.g. other servers to follow for the federation magic to happen.

**Reply by mastodonLike** [reply]
Cheers

**Comment by mastodonLike** [comment]
I have followed one of the listed server, But it says A message with a confirmation link has been sent to your email address. Please follow the link to activate your account. Please check your spam folder if you didn't receive this email.
................... There no link been sent so far !..................................

**Reply by MrGeneration** [reply]
It may take time for the mail to arrive depending on how busy the instance is. Also email sendout may take time in general. Greylisting and other anti spam measurements may strike. Hopefully arrives soon though.

**Reply by mastodonLike** [reply]
fab

**Comment by mastodonLike** [comment]
Hello again and sorry, I had email to confirm my details which I did, but the system asked for the current password and the new password , which I have provide, But keep saying invalid password !!!
I have insert all the correct details !

**Reply by MrGeneration** [reply]
That's always displayed.
You don't need to provide anything on that stage unless you either want to change your password or email address.

If you've verified your account you'll be able to go "back to Mastodon" which will throw you into the web app. 
Right now you're in your accounts settings and have various options.

![image](https://user-images.githubusercontent.com/6549061/200698181-da818efe-ec7f-43a9-a36d-7cb7a0f5b8a6.png)

**Reply by mastodonLike** [reply]
It has been verified thank you, but for the Profile metadata link and profile what exactly need to do with them please E.g ?

**Reply by MrGeneration** [reply]
> but for the Profile metadata link and profile what exactly need to do with them please

I'm sorry I fail to follow.
What's confusing you or what are you trying to do?
